client/tracing/Cargo.toml

```diff
@@ -19,10 +19,11 @@ rustc-hash = "1.1.0"
 serde = "1.0.101"
 serde_json = "1.0.41"
 slog = { version = "2.5.2", features = ["nested-values"] }
-tracing-core = "0.1.7"
+tracing = "0.1.18"
+tracing-subscriber = "0.2.10"
 sp-tracing = { version = "2.0.0-rc2", path = "../../primitives/tracing" }
 
 sc-telemetry = { version = "2.0.0-rc5", path = "../telemetry" }
 
 [dev-dependencies]
-tracing = "0.1.10"
+tracing = "0.1.18"
```

Comment:
Unnecessary, `tracing` is now a regular dependency already.

Reply:
good catch, removed